src/main/webapp/app/shared/tree-editor/tree-editor.component.html

```diff
@@ -17,7 +17,7 @@
 -->
 <div *ngFor="let concept of conceptList">
     <ul *ngIf="concept.parent === parentNotation" class="concept-tree ul-level-{{level!}}">
-        <li [attr.id]="!concept.deprecated ? 'code_' + concept.notation?.split('.')?.join('') : null" [ngClass]="{'y-bgr': !concept.title}">
+        <li [attr.id]="!concept.deprecated ? 'code_' + concept.notation.split('.').join('') : null" [ngClass]="{'y-bgr': !concept.title}">
```

Comment:
Wouldn't this throw an error if 'concept.notation' is undefined?

Reply:
`concept.notation` is defined to never be undefined in [src/main/webapp/app/shared/model/concept.model.ts](https://github.com/cessda/cessda.cvs.two/blob/651d4d39523e68e131ec6b4543ae67efbe9f753a/src/main/webapp/app/shared/model/concept.model.ts) so the check isn't necessary